src/opts.rs

```diff
@@ -7,7 +7,7 @@ use clap::{Parser, ValueHint};
 ///
 /// Supported formats: tar, zip, bz/bz2, gz, lz4, xz/lz/lzma, zst.
 ///
-/// Repository: https://github.com/ouch-org/ouch
+/// Repository: <https://github.com/ouch-org/ouch>
```

Comment:
Because this doc comment is used in both the `clap`-generated help output and the `rustdoc` generated documentation, this results in the angle brackets showing up in the `clap` output as well.

In my opinion this is something that would be nice to get fixed by `clap`, so I started an issue over there as well: [clap-rs/clap issue 3019](https://github.com/clap-rs/clap/issues/3019#issue-1052332655) although it doesn't have high priority there (totally understandable)

If having these angle brackets in the help output bothers you, we could try to duplicate this doc comment, using the doc comment itself for `rustdoc` and providing the long help for clap via `#[clap(long_about = "")]`

Reply:
> his results in the angle brackets showing up in the clap output as well.

I don't think this is worth it to remove rustdoc warnings since it isn't really used as a library and pretty much everyone use the help function

> If having these angle brackets in the help output bothers you, we could try to duplicate this doc comment, using the doc comment itself for rustdoc and providing the long help for clap via #[clap(long_about = "")]

maybe we can also just try to separate them and not have rustdoc touch any of it if it is possible. I don't see a reason this would be useful in a library anyways. Duplicating text isn't really an elegant solution

Reply:
Yeah you're probably right. I'll revert the URL changes and add an `#![allow(rustdoc::bare_urls)]` in main.rs

Reply:
I hope it's okay that I force-pushed this change, didn't want to have 2 separate commits for such a tiny change